01 - Community/People/apolaine.md

```diff
@@ -36,10 +36,9 @@ publish: true
 ### Unlisted plugins
 -->
 
-
+<!--
 ### Others
-
-- [[YouTube|YouTube Playlist]] of [Obsidian for DMing](https://www.youtube.com/playlist?list=PLvLJHmYS4OnXZIvguaL6rnvkIbimzVqaI) ^052eef
```

Comment:
Same with this link. If yes, I guess it's a decision of which method fits better for discovering the content: in note vs backlinks.

Reply:
Reverted this change since Apolaine never followed up with me.